src/components/numbers/Numbers.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+const Numbers = (props) =>{
+    let start = props.from;
+    let end = props.to;
+    const nums = Array(end-start+1)
+        .fill()
+        .map(() => start++);
+
+    if (props.odd){
+        return <ul>
+            {nums.filter(num => num % 2 ).map((num, index) => <li key={index}>{num}</li>)}
+        </ul>
+    }
+    if (props.even){
+        return <ul>
+            {nums.filter(num => num % 2 === 0).map((num, index) => <li key={index}>{num}</li>)}
```

Comment:
this `(num, index) => <li key={index}>{num}</li>` is used in every case, move it to one place